### PR TITLE
Adding ARIA attributes to toggle buttons

### PR DESCRIPTION
**Pull request by ivdimova**
For #278

### CODE REVIEW

**Comment by rianrietveld** [comment]
Hey @johnbillion and @ivdimova 
I think we need `aria-expanded` on the button instead of `aria-pressed`, to tell the screen reader what the `+` means: the content can be expanded and also tell the state. Like we do for hamburger menus. 
Toggle `aria-expanded="false"` and `aria-expanded="true"`

You could also add [aria-controls](https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions) to link the expandable content to the button.
But the screen reader support for that is questionable. http://www.heydonworks.com/article/aria-controls-is-poop


Also I would suggest extra text on the button (hidden by the 'screen-reader-text' class), to make the button text understandable, not just a `+`

Button list VoiceOver:
Button list VoiceOver

**Comment by ivdimova** [comment]
@rianrietveld @johnbillion this has been changed to aria-expanded, please check it this is ok.

File: output/Html.php [hunk 1]
```diff
@@ -146,7 +146,7 @@ protected function build_sorter() {
 	 * @return string Markup for the column sorter controls.
 	 */
 	protected function build_toggler() {
-		$out = '<button class="qm-toggle" data-on="+" data-off="-" aria-pressed="false">+</button>';
+		$out = '<button class="qm-toggle" data-on="+" data-off="-" aria-expanded="false">+<span class="screen-reader-text">Toggle button</span></button>';
```
Comment (rianrietveld) [inline output/Html.php]: Hey @ivdimova, add a space before the Toggle like `<span class="screen-reader-text"> Toggle button</span>`
That way it's pronounces better.
Then It's ok for me.

The text still isn't very unique, but it's better than it was.

Comment (johnbillion) [reply output/Html.php]: Thanks Rian. @ivdimova Can you also add the missing i18n that we just chatted about?

**Comment by ivdimova** [comment]
Ok to merge now @johnbillion?

**Pull request merged**